include/telnet_func.hpp

```diff
@@ -0,0 +1,83 @@
+#include <Arduino.h>
```

Comment:
Warum als `.hpp` und nicht `src/*.cpp`?
Wir sollten langsam schauen, dass wir Include-files wirklich nur noch für die Schnittstellen benutzen und für die Funktionen C Dateien nehmen.

Reply:
Hier würde ich allerdings warten, denn ich wollte das Funktionsmonstrum Uhr_func.h gerne aufspalten in zwei Sachen. Wordarray Funktionen und LED Funktionen. In diesem Zusammenhang könnte man die /src und /inc auch umstellen.